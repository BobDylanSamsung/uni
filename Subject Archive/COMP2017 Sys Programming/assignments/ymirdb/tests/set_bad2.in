set 12a 1 2 3
set a 1 2 3
list entries
set a 4 4 5
list entries
set b a
list entries
set b 1 2 c
list entries
bye
