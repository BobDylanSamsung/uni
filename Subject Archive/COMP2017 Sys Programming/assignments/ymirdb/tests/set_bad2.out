key not valid

ok

a [1 2 3]

ok

b [a]
a [4 4 5]

no such key

b [a]
a [4 4 5]

bye
